athena-jdbc/etc/test-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "athena_work_group" : "FederationIntegrationTests", /* The Athena Workgroup used for running integration tests (default: FederationIntegrationTests) */
+  "secrets_manager_secret" : "<secret name>",         /* Secret name used to retrieve user credentials from SecretsManager. */
+  "environment_vars" : {                  /* Parameters used by the connector's internal logic */
+    "spill_bucket" : "<spill bucket>",    /* The S3 bucket used for spilling excess data */
+    "spill_prefix" : "athena-spill",      /* The prefix within the S3 spill bucket (default: athena-spill) */
+    "disable_spill_encryption" : "false"  /* If set to true encryption for spilled data is disabled (default: false) */
+  },
+  "vpc_configuration" : {                 /* VPC configuration for DB instances within a VPC */
+    "vpc_id": "<VPC Id>",                 /* The VPC Id (e.g. vpc-xxx) */
+    "security_group_id": "<SG Id>",       /* The Security Group Id (e.g. sg-xxx) */
+    "subnet_ids": ["<Subnet 1>", "<Subnet 2>"],     /* A list consisting of at least one Subnet Id (e.g. subnet-xxxx) */
+    "availability_zones": ["<Zone 1>", "<Zone 2>"]  /* A list consisting of at least one AZ (e.g. us-east-1a) */
+  },
+  "user_settings" : {                     /* User customizable settings */
+    "redshift_db_name": "<DB Name>",      /* Name of the Db used by the Redshift integration tests */
```

Comment:
If parameters for one of the db's are not passed will the tests fail or just be ignored?

Reply:
So, the idea with the config files is that the tests won’t work out-of-the-box for Cx using the SDK w/o prior intervention (per Anthony). Since the config files are just empty templates, the invalid tests will be skipped as the test classes cannot be instantiated. This is probably the best behavior since provisioning the EC2 instance/cluster is very time consuming. So, it's better to fail up-front rather than spend 10 minutes for the cluster to come up just for the tests to fail. 

The idea is to replace the config files with valid ones on-the-fly when we automate the execution of the tests.